pycti/utils/opencti_stix2.py

```diff
@@ -198,7 +233,7 @@ def import_bundle_from_json(
         json_data: Union[str, bytes],
         update: bool = False,
         types: List = None,
-        retry_number: int = None,
+        work_id: str = None,
```

Comment:
there is a call in test (test_stix_crud.py) that has not been updated and still calls with a number : 
`bundles_sent = api_stix.import_bundle_from_json(split_bundle, False, None, 0)`